docs/manual/guides/grid-system.de.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Grid-System Einführung"
+description: "Ein Grid-System verstehen und einsetzen."
+url: "anleitungen/grid"
+aliases:
+    - /de/anleitungen/grid/
+    - /anleitungen/grid/
+weight: 54
+---
+
+Ein beliebtes Stilmittel ist die inhaltliche Aufteilung in Spalten. Hierbei hat sich der Einsatz von »Grid-Systemen«
+etabliert. Der zur Verfügung stehende Bereich wird in eine feste Anzahl von Spalten (Grids) unterteilt. Ein 
```

Comment:
etabliert. Der zur Verfügung stehende Bereich wird in eine feste Anzahl von Spalten (Columns) unterteilt. Ein